safekeeper/src/wal_backup_partial.rs

```diff
@@ -372,38 +416,21 @@ pub(crate) fn needs_uploading(
 ///
 /// When there is nothing more to do and the last segment was successfully uploaded, the task
 /// returns PartialRemoteSegment, to signal readiness for offloading the timeline.
-#[instrument(name = "Partial backup", skip_all, fields(ttid = %tli.ttid))]
+#[instrument(name = "partial_backup", skip_all, fields(ttid = %tli.ttid))]
```

Comment:
FYI we also have "broker pull", "WAL backup" and "recovery task" task names, it's probably better to rename them all in a separate PR.

Reply:
Made all of them snake case in separate commit.